build.rs

```diff
@@ -62,6 +62,27 @@ fn build_nasm_files() {
   rerun_dir("src/ext/x86");
 }
 
+#[cfg(feature = "asm")]
+fn build_asm_files() {
+  use std::fs::File;
+  use std::io::Write;
+  let out_dir = env::var("OUT_DIR").unwrap();
+  {
+    let dest_path = Path::new(&out_dir).join("config.h");
+    let mut config_file = File::create(dest_path).unwrap();
```

Comment:
Here are a few suggestions for consistency with the nasm integration.
The `PRIVATE_PREFIX` define can be merged into the generated `config.h`.
```suggestion
    let mut config_file = File::create(dest_path).unwrap();
    config_file.write(b" #define PRIVATE_PREFIX rav1e_\n").unwrap();
```